hq_superset/commands.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import json
+
+
+def subscribe_data_sources(superset_base_url, hq_base_url, data_sources_file_path, username, apikey):
```

Comment:
Hey @Charl1996 

Can you share more about whose username / apikey this is?
Are we using the same one for all domains?

I assumed we would use the domain's credentials like done in https://github.com/dimagi/commcare-analytics/blob/fb979e70b503e88cb101c66d8f52add281ce3a12/hq_superset/services.py#L173

Reply:
@mkangia 
> Can you share more about whose username / apikey this is?

Any superuser. 

Since a dev is running this command I thought it's reasonable to expect the dev to be able to generate and use the superuser credentials. 

> I assumed we would use the domain's credentials like done in

Initially I used it by found out soon enough that there's problems with using the `HQRequest` class with respect to the auth token. The class relies on the fact that there's an active user request which is oauth authenticated. Since running a script manually won't have this context I decided to take this approach in this PR.

Reply:
We offlined about this and decided to skip re-subscription to avoid using a dev api key for this and additionally to have subscription authority with users/projects only.

Reply:
:+1: @Charl1996 does that mean that this PR will be closed?

Reply:
@kaapstorm Thanks for flagging. Uhm...do we see any future use for something like this?